metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/ObjectsImporter.java

```diff
@@ -221,9 +238,28 @@ public ReferenceSequence importReferenceSequence(String accession) {
         return referenceSequence;
     }
 
-    private Taxonomy extractTaxonomyFromAssembly(AssemblyType assemblyType) {
-        AssemblyType.TAXON taxon = assemblyType.getTAXON();
-        return new Taxonomy(taxon.getTAXONID(), taxon.getSCIENTIFICNAME());
+    private ReferenceSequence retrieveReferenceSequenceFromEntrez(String accession) throws Exception {
+        String idRetrievalUrl = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch" +
+                ".fcgi?db=assembly&term={accession}";
+        String assemblyRetrievalFromIdUrl = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary" +
+                ".fcgi?db=assembly&id={id}";
+        final String idStartTag = "<Id>";
+        final String idEndTag = "</Id";
+        Map<String, String> uriVariables = new HashMap<>();
+        uriVariables.put("accession", accession);
+
+        if (entrezApiKey != null && !entrezApiKey.isEmpty()) {
+            final String apiKeyQueryString = "&api_key={entrezApiKey}";
+            uriVariables.put("entrezApiKey", entrezApiKey);
+            idRetrievalUrl = idRetrievalUrl + apiKeyQueryString;
+            assemblyRetrievalFromIdUrl = assemblyRetrievalFromIdUrl + apiKeyQueryString;
+        }
+
+        String idXml = sraxmlRetrieverByAccessionForceApi.getXml(idRetrievalUrl, uriVariables);
```

Comment:
Why is this now querying the ENA API?

Reply:
I was providing the custom entrezUrl, but upon realizing the confusion. I would change as suggested in https://github.com/EBIvariation/metadata-ws/pull/62#discussion_r307808897